core/src/test/java/org/apache/accumulo/core/metadata/schema/TabletMetadataTest.java

```diff
@@ -203,6 +207,7 @@ public void testAllColumns() {
     assertEquals(unsplittableMeta, tm.getUnSplittable());
     assertEquals(ecMeta.toJson(), tm.getExternalCompactions().get(ecid).toJson());
     assertEquals(10, tm.getFlushNonce().getAsLong());
+    assertEquals(tsi, tm.getMigration());
```

Comment:
Would be nice if tests like this and `AmpleConditionalWriterIT` failed when new columns are added to the the metadata tables but are not tested. This could be a follow on. Would also have to ensure all existing columns are tested.